src/test/java/ma/boot/springboot/ReadingCsvFileTests.java

```diff
@@ -0,0 +1,35 @@
+package ma.boot.springboot;
+
+import java.io.IOException;
+import ma.boot.springboot.service.read.ReadingCsvFile;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ReadingCsvFileTests {
+    private static final String CORRECT_CSV_FILE = "three_records.csv";
+    private static final String NOT_EXISTED_FILE = "file123.csv";
+    private static final String WRONG_DATA = "incorrect_data.csv";
+    private static final String EMPTY_FILE = "empty_file.csv";
+
+    @Test
+    public void loadCorrectFile() throws Exception {
+        int result = ReadingCsvFile.readWithCsvBeanReader(CORRECT_CSV_FILE);
```

Comment:
what is result and 3 - it's not clear for me in this code

Reply:
Number of records which were read. I have already changed this in SB2 - I am creating a list with read records